dev/system-test/firestore.ts

```diff
@@ -194,6 +196,126 @@ describe('Firestore class', () => {
   });
 });
 
+describe('CollectionGroup class', () => {
+  const desiredPartitionCount = 3;
+  const documentCount = 2 * 128 + 127; // Minimum partition size is 128.
```

Comment:
Btw, just caught this, but why is the minimum partition size 128? I didn't see this requirement elsewhere in the code. Also, is this something the developer needs to know?

Reply:
This is not the partition size, but rather the minimum result set size in order for partitions to work. If a collection group has fewer than 128 documents, then it will not be partitioned. This is deemed acceptable with the current API as the provided partition count value is documented to be best effort.